providers/base/units/wwan/test-plan.pxu

```diff
@@ -39,25 +39,3 @@ include:
     after-suspend-wwan/gsm-connection-.*-auto
 bootstrap_include:
     wwan_resource
-
-id: wwan-manual
-unit: test plan
-_name: Manual wwan tests
-_description: Manual wwan tests for Snappy Ubuntu Core devices
-include:
-    wwan/detect-manual
-    wwan/gsm-connection-manual
-    wwan/check-sim-present-manual
-    wwan/scan-networks-manual
-    wwan/gsm-connection-interrupted-manual
```

Comment:
Same as in a previous comment, you can just delete these line after `include:` but keep the rest of the `wwan-manual` test plan.

I noticed you deleted some of the jobs completely (`wwan/gsm-connection-manual`, `wwan/check-sim-present-manual`) but left others (`wwan/detect-manual`, ...). Is there a good reason for this?

Reply:
wwan/scan-networks-manual and wwan/gsm-connection-interrupted-manual are not covered in the automated test plan, these two should not be removed.
In addition to that the 2 jobs mentioned above still depend on wwan/check-sim-present-manual which has been removed in this PR, that will be an issue.
Some rework may have to be done on wwan/scan-networks-manual and wwan/gsm-connection-interrupted-manual to adapt the changes.